{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -6,25 +6,15 @@ skipsdist = True
 
 [testenv]
 setenv =
-    DJANGO_SETTINGS_MODULE={{ cookiecutter.project_slug }}.settings.base
+    DJANGO_SETTINGS_MODULE = {{ cookiecutter.project_slug }}.config.settings.test
     PYTHONPATH={toxinidir}
-    SECRET_KEY=test
     DATABASE_URL=postgresql://{{ cookiecutter.project_slug }}:{{ cookiecutter.project_slug }}@localhost/{{ cookiecutter.project_slug }}
-deps = -r{toxinidir}/requirements/base.txt
+deps = -r{toxinidir}/requirements/dev.txt
 whitelist_externals = scripts/check_migrations.sh
 commands=
     scripts/check_migrations.sh
+    pytest --cov-report=term --cov={{ cookiecutter.project_slug }} --cov=apps -n 2
 
-[testenv:py34]
-deps =
-    {[testenv]commands}
-    django-admin.py test --noinput []
-
-[testenv:py34-coverage]
-deps =
-    {[testenv]deps}
-    coverage
-commands =
-    coverage erase
-    coverage run --source="." manage.py test --noinput []
-    coverage report --omit=".tox/*,config/wsgi.py,config/settings/*,fabfile.py"
+[pytest]
+DJANGO_SETTINGS_MODULE = {{ cookiecutter.project_slug }}.config.settings.test
```

Comment:
Although first level in the ini file is with whitespace, for example
`envlist = py34`